home/tests/test_insert_question_feature.py

```diff
@@ -99,20 +99,22 @@ class TestViews:
             # User entered all the required fields and sub-subject
             ({'title': "Question in Math",
               'subject': 1, 'grade': Grade.GRADE7, 'book': 2}),
-            # User entered all the required fields and sub-subject and book
+            # User entered all the required fields, sub-subject and book
             ({'title': "Question in Math",
               'subject': 1, 'grade': Grade.GRADE7, 'book_page': 23}),
             # User entered all the required fields and a valid book page
         ])
         @pytest.mark.django_db
-        def test_post_valid_question_with_client(self, client, valid_data):
+        def test_post_valid_question_with_client(self, client, valid_data, authenticated_user):
             with pytest.raises(Question.DoesNotExist):
                 assert Question.objects.get(title=valid_data["title"])
 
-            client.login(username='Lior', password='LiorLior')
-            client.post('/explore/new_question', data=valid_data)
+            response = client.post('/explore/new_question', data=valid_data)
 
             assert Question.objects.filter(title=valid_data["title"]).exists()
+            assert response.status_code == 302
+            question_id = Question.objects.filter(title=valid_data["title"])[0].id
```

Comment:
@avivz450 
I think you can use "get" instead of "filter" and [0]. 
I mean you can write 
` question_id = Question.objects.get(title=valid_data["title"]).id`
instead of 
` question_id = Question.objects.filter(title=valid_data["title"])[0].id`

Reply:
Good idea, done